Feature/home page

VishnuMarontavida opened the pull request:
Completed the task on the feature module 'feature/home-page'

VishnuMarontavida committed:
Completed the Home page based on the user story

VishnuMarontavida committed:
Merge branch 'main' into feature/home-page

VishnuMarontavida committed:
Fixed issue for multiple actions from an effect

VishnuMarontavida committed:
Removed the unwanted commented area

MarcoHuib commented:
Please don't use style="" but use the bootstrap classes as much if possible.

Some handy links:
 https://getbootstrap.com/docs/5.0/utilities/spacing/

VishnuMarontavida committed:
changed styles respective of bootstrap class

VishnuMarontavida commented:
> Please don't use style="" but use the bootstrap classes as much if possible.
> 
> Some handy links: https://getbootstrap.com/docs/5.0/utilities/spacing/

Changed the inline styles in html page and added the bootstrap class

VishnuMarontavida committed:
Removed the inline style in header

MarcoHuib reviewed (approved):
Lgtm!
